OpenRA.Mods.Common/UpdateRules/UpdatePath.cs

```diff
@@ -74,6 +74,7 @@ public class UpdatePath
 				new RenameSpins(),
 				new CreateScreenShakeWarhead(),
 				new RenameRallyPointPath(),
+				new MoveClassicFacingFudge(),
```

Comment:
This path now corresponds to the prep-2003 branch - #17808 adds a new "playtest-20200303" and puts the new rules outside a dated folder until we know the final release date.

We will need to either do the same here, or rebase on top of it after it has been merged.